deploy/kubernetes/webhook-example/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: filestorecsi-validation
-        image: gcr.io/saikatroyc-test/gcp-filestore-csi-driver-webhook:v2
+        image: gcr.io/leiyi-k8s-testing/gcp-filestore-csi-driver-webhook:v0.1
```

Comment:
Put a comment here to replace with appropriate image.

Reply:
done